SpliceRegion.pm

```diff
@@ -134,12 +141,29 @@ sub run {
         #   G => '_to_purine',
         # }
       },
-    );
+    ) unless $extended_flag;
+    
+    @terms = (
+      {
+        term => 'extended_intronic_splice_region_variant_5prime',
+        region => [$intron->{$donor_coord}, $intron->{$donor_coord} + (9 * $strand_mod)]
```

Comment:
Could it be better to put "9" into a variable ?

Reply:
Yeah, probably - I just followed the convention previously used in the file for the other terms and didn't think it would be worth the rewrite for all of them